v2/loader/loader.go

```diff
@@ -225,6 +233,21 @@ func (tl *TypeLoader) LoadColumns(typeTpl *models.Type) error {
 	}
 
 	columnTypes := tl.tableCustomTypes(typeTpl.TableName)
+
+	// validate custom type columns
+	if columnTypes != nil {
+		columnSet := map[string]bool{}
```

Comment:
super nit: good to use `map[string]struct{}` when the value is meaningless, then substitute `struct{}{}` instead of `true`